examples/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
```

Comment:
What to do with this file?

Reply:
Ah sorry about that :see_no_evil: I accidentially added some files that were around in my folder. Fixed and rebased.